bot/handlers/not_sending_videos/search_handler.py

```diff
@@ -16,18 +17,28 @@
     get_log_search_results_sent_message,
 )
 from bot.search.transcription_finder import TranscriptionFinder
+from bot.settings import settings
 
 
 class SearchHandler(BotMessageHandler):
     def get_commands(self) -> List[str]:
         return ["szukaj", "search", "sz"]
 
-    async def _do_handle(self, message: Message) -> None:
+    async def is_any_validation_failed(self, message: Message) -> bool:
         content = message.text.split()
         if len(content) < 2:
-            return await self._reply_invalid_args_count(message, get_invalid_args_count_message())
+            await self._reply_invalid_args_count(message, get_invalid_args_count_message())
+            return True
 
         quote = " ".join(content[1:])
+        if not await DatabaseManager.is_admin_or_moderator(message.from_user.id) and len(quote) > settings.MAX_SEARCH_QUERY_LENGTH:
+            await message.answer(get_message_too_long_message())
+            return True
+
+        return False
```

Comment:
już nie wiem gdzie tego szukać ale chyba robiłeś na to funkcję nie

Reply:
na to jeśli mi się nic nie pojebało od ostatniego tygodnia i dobrze teraz szukam to chyba nie.
chyba że ci chodzi żeby to do is_any_validation_failed() tylko takich checków to nwm jak miałbym do tego wywalić jak to się musi wykonać w określonym momencie w do handle